pkg/azureconstants/azure_constants.go

```diff
@@ -63,6 +63,7 @@ const (
 	PvcNameTag                    = "kubernetes.io-created-for-pvc-name"
 	PvNameKey                     = "csi.storage.k8s.io/pv/name"
 	PvNameTag                     = "kubernetes.io-created-for-pv-name"
+	PodKey                        = "disk.csi.azure.com/podName"
```

Comment:
Is the use case for this different than `PodNameKey` below? It should be `disk.csi.azure.com/pod-name` for consistency, but that would duplicate the constant below.

Reply:
I totally forgot that we had `podNameKey`. Will switch to that.